### PR TITLE
Views

### PR DESCRIPTION
**Pull request by ericmahare**
In this section I did the following : -

- [ ] Used provided wireframe to implement the users and posts views

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1 +1,26 @@
...
+              <p><%= post.text %></p>
+            </div>
+              <h4><span>Comment <%= post.commentscounter %></span> <span>|</span> <span>Likes <%= post.likescounter %></span></h4>
+        </div>
+      <% end %>
+      <% @commets.each do |comment|%>
+      <div>
+        <p><%=comment.Text%></p>
+      </div>
+      <%end%>
```
Comment (zainsadaqat) [inline app/views/posts/index.html.erb]: All post comments should be displayed on the show page. You may use the recent_comments methods created in previous activities to display recent comments on index page
### For Example
```ruby
<% post.recent_comments.each do |comment| %>
  <p>
    <%= comment.author.name %>: <%= comment.text %>
  </p> 
<% end %>
```

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Required Changes 🔁
# Hello @ericmahare  😄 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors
  ✅  Descriptive Pull Request

## Required Changes ♻️

_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/index.html.erb (hunk 1 above)
Comment (ericmahare) [reply app/views/posts/index.html.erb]: Thanks for your review let me work on the changes

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @ericmahare👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**